cmd/lncli/commands.go

```diff
@@ -745,14 +746,14 @@ func create(ctx *cli.Context) error {
 	defer cleanUp()
 
 	fmt.Printf("Input wallet password: ")
-	pw1, err := terminal.ReadPassword(0)
+	pw1, err := terminal.ReadPassword(int(syscall.Stdin))
```

Comment:
Does this have to be cast to `int`? At least on Unix it does not, but maybe on Windows?

Reply:
Looks like Windows needs the cast:
```
cmd\lncli\commands.go:749:35: cannot use syscall.Stdin (type syscall.Handle) as type int in argument to terminal.ReadPassword
```

Reply:
Ok, thanks!